### PR TITLE
task/DES-1670 - Update MyData in WorkSpace Data Browser to match Data Depot

### PR DESCRIPTION
**Pull request by fnets**
The sort for objects in MyData in the Workspace Data Browser now matches the sort in the Data Depot.

### CODE REVIEW

File: designsafe/static/scripts/workspace/html/data-browser.html [hunk 1]
```diff
@@ -63,28 +63,49 @@ <h2>Data Depot Browser</h2>
...
-                <tr ng-if="data.loading"><td colspan="3"><p class="lead"><i class="fa fa-spin fa-circle-o-notch"></i> Loading files...</p></td></tr>
-                <tr ng-if="data.error"><td colspan="3"><p class="alert alert-danger">{{data.error}}</p></td></tr>
-                <tr ng-if="data.busyListingPage">
-                  <td>
-                        <span class="bg-info text-info">
-                           <i class="fa fa-spinner fa-spin"></i> Loading...
-                        </span>
-                  </td>
-                </tr>
+                    <tr ng-if="data.filesListing.system !== 'nees.public' && data.filesListing.system === 'designsafe.storage.projects'"
```
Comment (rstijerina) [inline designsafe/static/scripts/workspace/html/data-browser.html:66]: Can get rid of the `nees.public` term here
```suggestion
                    <tr ng-if="data.filesListing.system === 'designsafe.storage.projects'" 
```

**Comment by rstijerina** [comment]
What's the difference between this PR and #766?

File: designsafe/static/scripts/workspace/html/data-browser.html [hunk 2]
```diff
@@ -63,28 +63,49 @@ <h2>Data Depot Browser</h2>
...
+                                ds-data-draggable
+                                data-ds-data="agave://{{f.system}}/{{f.path}}">
+                                <i class="fa {{f.icon()}}"></i>
+                                <span>{{renderName(f)}}</span>
+                            </a>
+                        </td>
+                        <td>{{f.length|bytes:0}}</td>
+                    </tr>
+                    <tr ng-if="data.filesListing.system !== 'nees.public' && data.filesListing.system !== 'designsafe.storage.projects'"" 
+                        ng-repeat="f in data.filesListing.children | orderBy : ['name']">
```
Comment (fnets) [inline designsafe/static/scripts/workspace/html/data-browser.html:84]: @rstijerina this is the difference between this PR and #760

File: designsafe/static/scripts/workspace/html/data-browser.html (hunk 1 above)
Comment (fnets) [reply designsafe/static/scripts/workspace/html/data-browser.html]: Done

**Pull request merged**